src/services/AudiusBackend.js

```diff
@@ -662,9 +662,15 @@ class AudiusBackend {
         account._artist_pick = body.pinnedTrackId || null
         account.twitterVerified = body.twitterVerified || false
         account.instagramVerified = body.instagramVerified || false
+      } catch (e) {
+        console.error(e)
+      }
+      try {
+        const userBank = await audiusLibs.solanaWeb3Manager.getUserBank()
```

Comment:
is it necessary to wait for this on account load? this makes solana RPC part of the every page load deal. at the very lease, maybe we could do some smart promise.all?

Reply:
sanity check me on this one @jowlee but I think this is actually fine - getUsersBank() is only async bc the call to publicKey.createWithSeed is async due to the bump seed approach, and the Solana client shouldn't be doing any async work on init. Could use a comment

Reply:
Yes, you're right @piazzatron, it shouldn't make any rpc calls. 
To get the user bank, it takes in the eth address, and a couple solana constants and encodes it into a solana public key